CSS에 가이드 색상 추가

min6choi opened the pull request:
- css에 구글 가이드 색상 추가했습니다. 
- css랑 sass를 vscode로 테스트하려는데 live server가 작동이 안하길래 일단 색상표만 상단에 입력해 놨습니다. (예전에 개인 jekyll 만들었을 때는 됐던걸로 기억하는데 이 블로그는 안되네요 ㅠㅠ 혹시 다른 분들은 어떻게 확인하시나요?) 
- 수빈님 가이드라인대로 pull request했는데 혹시 잘못한 거 있으면 알려주세요 ㅠ 다 까먹었네요...

min6choi committed:
CSS에 가이드 색상 추가

- css에 구글 가이드 색상 추가

MOBUMIN reviewed (commented):
저는 css, sass 넣어서 jekyll serve 잘 됩니다 ㅠㅠ
PR 잘 날리셨어요 :)
:root 이런 태그는 처음 봤는데, 전역으로 사용할 수 있게 해주는 거였군요. 배워갑니다!
컬러 팔레트 등록 하고 어디에 어떻게 쓰실건가용?

min6choi commented:
> 저는 css, sass 넣어서 jekyll serve 잘 됩니다 ㅠㅠ
> PR 잘 날리셨어요 :)
> :root 이런 태그는 처음 봤는데, 전역으로 사용할 수 있게 해주는 거였군요. 배워갑니다!
> 컬러 팔레트 등록 하고 어디에 어떻게 쓰실건가용?

지금 블로그 색상이 예를들면 초록색만 3가지 이상을 사용하고 있어서, 색상을 최소화하고 정돈되어 보이게 일단 메인페이지 색상만 적용해보려고요!

MOBUMIN reviewed (dismissed):
LGTM!

min6choi committed:
css 앞부분 색상 변경 test

회색을 추가하고
page number의 색상을 테스트로 변경해보았습니다.

MOBUMIN reviewed (commented):
혹시 어떤 부분에서 어떻게 바뀌어야 맞는건지 스크린샷 첨부해주실 수 있나용 ?
코드상 green으로 색이 바뀌어야되는 것 같은데, 저는 검정색으로 바뀌어서요 !

min6choi committed:
blue base로 재변경

link 가능한 텍스트 색상을 가이드 색상에 맞춰 변경했습니다.

min6choi commented:
> 혹시 어떤 부분에서 어떻게 바뀌어야 맞는건지 스크린샷 첨부해주실 수 있나용 ?
> 코드상 green으로 색이 바뀌어야되는 것 같은데, 저는 검정색으로 바뀌어서요 !

지난번 test로 작성할 때 색상 코드를 잘못 입력해서 검정색으로 바뀌었어요 ;;
이번에는 제대로 변경했습니다. 참고로 시립대와 어울리는 파란색으로 변경했습니다. (+사진도 첨부했어요!)

<img width="540" alt="Screenshot 2021-05-12 at 9 02 58 PM" src="https://user-images.githubusercontent.com/57276820/117972013-90f55c80-b365-11eb-8ea8-dec26ec7dd4b.png">

min6choi committed:
색상 변경

green -> blue
#fff 또는 흰색 -> white
#222 또는 검정색 -> black

about 프로필 border
green -> light black 으로 변경했습니다.

min6choi commented:
블로그 색상에 통일을 주기 위해 green 계열의 색상을 모두 blue로 바꾸었고, 
백색과 검정색은 가이드라인의 백색과 검정색으로 변경했습니다. 

그리고 about 프로필사진 border도  green->회색으로 변경했습니다. 
거의 변경된게 없어보일거라서 크게 변경된 부분은 사진 첨부해 올립니다. 

<img width="528" alt="Screenshot 2021-05-14 at 8 37 23 PM" src="https://user-images.githubusercontent.com/57276820/118266431-8b2e8100-b4f5-11eb-84d7-b726d68e47d3.png">

왼쪽 파란색 border가 변경된 부분입니다.

MOBUMIN reviewed (approved):
색상 적용 다 된거 확인했습니다 :)
저는 파스텔 느낌이 더 좋긴 한데, 이제 곧 여름이니까 이런 파랑파랑도 괜찮네욤 :))
다음에 theme 같은거 만들어서 summer, spring 이런식으로 색상 한번에 변경하는 기능도 좋을 것 같아요~

min6choi commented:
> 색상 적용 다 된거 확인했습니다 :)
> 저는 파스텔 느낌이 더 좋긴 한데, 이제 곧 여름이니까 이런 파랑파랑도 괜찮네욤 :))
> 다음에 theme 같은거 만들어서 summer, spring 이런식으로 색상 한번에 변경하는 기능도 좋을 것 같아요~

theme 만드는 건 처음 들어보는데 혹시 관련 링크 올려주실 수 있나요? 제가 한번 만들어보겠습니다!

MOBUMIN commented:
> > 색상 적용 다 된거 확인했습니다 :)
> > 저는 파스텔 느낌이 더 좋긴 한데, 이제 곧 여름이니까 이런 파랑파랑도 괜찮네욤 :))
> > 다음에 theme 같은거 만들어서 summer, spring 이런식으로 색상 한번에 변경하는 기능도 좋을 것 같아요~
> 
> theme 만드는 건 처음 들어보는데 혹시 관련 링크 올려주실 수 있나요? 제가 한번 만들어보겠습니다!

엇 저도 정확히 아는게 아니어서
그냥 config.yml에 공용 변수? 하나 만들어서 뭔가 다 적용시키는 추상적인 느낌이었어요.... 

https://github.com/mmistakes/jekyll-theme-basically-basic#skin

스킨이 적용된 jeykll thema인데 이런 git 레포지토리 참고하면 어떨까요 ?

LeeMir reviewed (approved):
LGTM!! 푸른 계열 좋네용

LeeMir commented:
제꺼랑 충돌날 것 같아서 바로 merge 하겠습니다!
